NightscoutServiceKit/RemoteCommands/V1/RemoteCommandSourceV1.swift

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
The 2 commands sources, for Remote 1.0 vs Remote 2.0, allow us to support both paradigms at the same time which will be needed eventually.